src/main/java/frc/utils/PathReader.java

```diff
@@ -0,0 +1,52 @@
+package frc.utils;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class PathReader {
```

Comment:
This entire class will never work at runtime, and seems very unneeded.
If the starting pose is required, there seems to be an [already existing method](https://github.com/mjansen4857/pathplanner/blob/3a7fc679bb48563145dbafd41fdde12fb29bb931/pathplannerlib/src/main/java/com/pathplanner/lib/commands/PathPlannerAuto.java#L54) in pathplanner that allows you to do this.

Reply:
We should remove this. As discussed in Slack yesterday, this isn't necessary if you provide a starting pose in the JSON files used by PathPlanner